sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/SensorContext.java

```diff
@@ -64,9 +64,22 @@
 
   /**
    * @since 5.5
+   * @deprecated since 6.0 replaced by {@link #getRuntimeApiVersion()}
    */
+  @Deprecated
   Version getSonarQubeVersion();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)